### ISSUE TITLE
`--idea` flag option not flexible

### ISSUE THREAD

**Issue by matheus-teles** [holgerbrandl/kscript#263]
Today the command:
```
kscript --idea script.kts
```
Does not work if intellij was installed through snap because it opens with `intellij-idea-community` command.

I was wondering if there could be an enhancement for it to receive an argument that explicits the command that open Intellij like:
```
kscript --idea=intellij-idea-community script.kts
```
or something to make it more flexible

**Comment by holgerbrandl** [comment]
Good point. Either like that or via an environment property. I typically prefer the latter for rarely used features. 

Would you be able/willing to contribute such a feature via PR?

**Comment by matheus-teles** [comment]
Yes, I will do this feature through an environment property via PR

Thanks for the invitation

**Comment by matheus-teles** [comment]
The PR is here #264

**Comment by jiraguha** [comment]
Hey sounds fun... It feels a bit strange to still call the helper `--idea` as I could launch any other command than intellij. @matheus-teles for example in your proposition I could do an `export KSCRIPT_IDEA_COMMAND=code` and it will run vscode

**Comment by holgerbrandl** [comment]
Fixed via PR in #264
